tests/E2E/OrganizationTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of the contentful-management.php package.
+ *
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\E2E;
+
+use Contentful\Management\Query;
+use Contentful\Tests\End2EndTestCase;
+
+class OrganizationTest extends End2EndTestCase
+{
+    /**
+     * @vcr e2e_organization_get_collection.json
+     */
+    public function testGetOrganizations()
+    {
+        $organizations = $this->client->getOrganizations();
+
+        $this->assertCount(3, $organizations);
+
+        $organization = $organizations[2];
```

Comment:
That was my problem with the initial PR as well 😅 
It **is** fragile, I agree, but I think that for the moment that shouldn't be a problem. I'm actually using a special user for testing purposes, which I think I'll update soon-ish, and then this test will be more robust 🙂 

The underlying problem with the organizations endpoint is that you can't easy filter values, unless you do it manually in your code. So I chose a pragmatic approach, and I will fix this as soon as I've stabilized a couple other things 👍